flipper/contrib/__init__.py

```diff
@@ -24,9 +23,15 @@
     "CachedFeatureFlagStore",
     "ConsulFeatureFlagStore",
     "MemoryFeatureFlagStore",
-    "PostgreSQLFeatureFlagStore",
     "RedisFeatureFlagStore",
     "ReplicatedFeatureFlagStore",
     "S3FeatureFlagStore",
     "ThriftRPCFeatureFlagStore",
 ]
+
+try:
+    from .postgresql import PostgreSQLFeatureFlagStore  # noqa
+
+    __all__.append("PostgreSQLFeatureFlagStore")
+except ModuleNotFoundError:
+    pass
```

Comment:
I'd prefer to do the conditional import in the `PostgreSQLFeatureFlagStore` so we don't have to do the import *.